content/docs/command-reference/checkout.md

```diff
@@ -158,10 +159,9 @@ baseline-experiment     <- First simple version of the model
 bigrams-experiment      <- Uses bigrams to improve the model
 ```
 
-We can now just run `dvc checkout` that will update the most recent `model.pkl`,
-`data.xml`, and other files that are tracked by DVC. The model file hash is
-defined in the `dvc.lock` file, and in the `data.xml.dvc` file for the
-`data.xml`:
+We can now run `dvc checkout` to update the most recent `model.pkl`, `data.xml`,
+and any other files tracked by DVC. The model file hash, `ab349c2...`, is saved
+in the `dvc.lock` file, for example, so it can be confirmed with:
```

Comment:
do we need `for example`? I think previous version was more or less fine, except `just` and other minor things

Reply:
Oops yes this is definitely out of scope sorry. Done before we discussed that matter.

But the previous paragraph didn't make much sense given the example presented below. Simplified it more now (removed "for example").